src/app/public/modules/split-view/types/split-view-message-type.ts

```diff
@@ -0,0 +1,3 @@
+export enum SkySplitViewMessageType {
+  FocusFirstItemInWorkspace = 0
```

Comment:
You might be able to get away with a simpler and more generic message type that is easier to remember, something like `FocusWorkspace`?

Reply:
Done.